config/clusters/2i2c/jackeddy.values.yaml

```diff
@@ -0,0 +1,181 @@
+basehub:
+  userServiceAccount:
+    annotations:
+      iam.gke.io/gcp-service-account: pilot-hubs-jackeddy@two-eye-two-see.iam.gserviceaccount.com
+  jupyterhub:
+    # https://infrastructure.2i2c.org/howto/features/ephemeral/#pre-pulled-images
+    prePuller:
+      continuous:
+        enabled: true
```

Comment:
I think we can't use pre-puller for this hub because it doesn't have a specific image it needs, but pangeo/pangeo-notebook + unlisted_choice.

If we have this, a new node starting for a user pod with unlisted_choice wanting to use image X can end up waiting on the image pulling of pangeo/pangeo-notebook.

Related:
- https://github.com/2i2c-org/infrastructure/issues/1854
- https://github.com/2i2c-org/infrastructure/issues/2344
- https://github.com/2i2c-org/infrastructure/issues/2541

Reply:
Technically, they only requested the pangeo image I believe (ah, and Heliocloud as another option, but that one was private 🤔 ), the unlisted_choice is something we assumed they will need.

But, yes, I will rm the pre-puller and add it later if need be. Thank you!